### ISSUE TITLE
Login error: Unexpected response, this might indicate a blocked IP.

### ISSUE THREAD

**Issue by IustinianCM** [instaloader/instaloader#1798]
Hello! Everything went well. But now, I have this error Login error: Unexpected response, this might indicate a blocked IP.
File "C:\Users\iusti\Desktop\Instaloader\lib\site-packages\instaloader\instaloadercontext.py", line 254, in login
    raise ConnectionException("Login error: Unexpected response, this might indicate a blocked IP.")
instaloader.exceptions.ConnectionException: Login error: Unexpected response, this might indicate a blocked IP.

If I use a VPN and change my IP, the error goes nowhere. The solution is to switch to another source, another ISP. But why?

**Comment by top31** [comment]
failed management
And a failed tool
To this day, nothing has been done. No update has been released
I will look for others
They apply inside the tool what they want, they don't care about anyone, damn them
the program Download images bulk is better than this trivial tool
Their frivolous tool
You download a few
Then you will end up at the rate limit
as they call deceptive liars

**Comment by top31** [comment]
> Hello! Everything went well. But now, I have this error Login error: Unexpected response, this might indicate a blocked IP. File "C:\Users\iusti\Desktop\Instaloader\lib\site-packages\instaloader\instaloadercontext.py", line 254, in login raise ConnectionException("Login error: Unexpected response, this might indicate a blocked IP.") instaloader.exceptions.ConnectionException: Login error: Unexpected response, this might indicate a blocked IP.
> 
> If I use a VPN and change my IP, the error goes nowhere. The solution is to switch to another source, another ISP. But why?

failed management
And a failed tool
To this day, nothing has been done. No update has been released
I will look for others
They apply inside the tool what they want, they don't care about anyone, damn them
the program Download images bulk is better than this trivial tool
Their frivolous tool
You download a few
Then you will end up at the rate limit
as they call deceptive liars

**Comment by tam-carre** [comment]
@top31 what have *you*_done?

**Comment by Phil-Acosta** [comment]
@IustinianCM Have you checked to see if you can sign-in via browser on the same computer?  

If you haven't, check that first.  If you find that Instagram has blocked your browser login then the Instaloader won't work either.  I've seen where Instagram can block browser/PC sign-in but still allow app sign-in on the same network.

**Comment by vicmac-github** [comment]
> @IustinianCM Have you checked to see if you can sign-in via browser on the same computer?
> 
> If you haven't, check that first. If you find that Instagram has blocked your browser login then the Instaloader won't work either. I've seen where Instagram can block browser/PC sign-in but still allow app sign-in on the same network.

@Phil-Acosta how long it can take to Instagram unblock you?

**Comment by i-cant-git** [comment]
I have the same problem, how can I debug?
I tried many accounts, I can login on my browser.

**Comment by MosEisley34** [comment]
Is there a workaround?

**Comment by aandergr** [comment]
Duplicate of #1635
